hknweb/candidate/admin.py

```diff
@@ -75,6 +64,23 @@ def csec_reject(self, request, queryset):
     csec_reject.short_description = "Mark selected as rejected (csec)"
 
 
+class BitByteActivityAdmin(admin.ModelAdmin):
+
```

Comment:
It may make admin lives easier if we had an apt name for this (maybe some combination of the candidate name and the first few words of the activity description?)
![hknalex bitbyte admin name_LI](https://user-images.githubusercontent.com/46059916/74601940-59abd900-5058-11ea-9a91-bdf8e489cdb4.jpg)

For example, something like this is more helpful and accessible:
![hknalex bitbyte admin name suggestion](https://user-images.githubusercontent.com/46059916/74601966-b3140800-5058-11ea-95d4-8ed78a1dfe4c.PNG)

Can maybe implement by creating `str` or `repr` functions for the [`BitByteActivity`](https://github.com/compserv/hknweb/pull/186/files?file-filters%5B%5D=.html&file-filters%5B%5D=.py#diff-f7910a17582b60486a0c61e7208c9eeeR60) class?

Reply:
Done